src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.diff.sql

```diff
@@ -17,8 +17,8 @@ CREATE TYPE dbo.StringSearchParamTableType_2 AS TABLE
 ALTER TABLE dbo.StringSearchParam
 ADD IsMin bit NOT NULL,
     IsMax bit NOT NULL,
-    CONSTRAINT IsMin_Constraint DEFAULT 0 FOR IsMin,
-    CONSTRAINT IsMax_Constraint DEFAULT 0 FOR IsMax;
+    CONSTRAINT string_IsMin_Constraint DEFAULT 0 FOR IsMin,
```

Comment:
>string_IsMin_Constraint [](start = 15, length = 23)

Shouldn't these changes be part of 8.sql as well?

Reply:
We need to only set the default here because we are adding two new non-null columns to an existing table/database. For brand new databases, we don't want to set the default value since the code will take care of that part. Hence the constraints are not added for 8.sql and only for 8.diff.sql